aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,37 @@
 ---
-title: Introduction to Razor Components
+title: Introduction to ASP.NET Core Razor Components
```

Comment:
I prefer "Introduction to Razor Components", so that this content can apply equally to Blazor and ASP.NET Core Razor Components.